src/aggregate/functions/string.c

```diff
@@ -299,6 +299,19 @@ static int stringfunc_split(ExprEval *ctx, RSValue *result, RSValue **argv, size
   return EXPR_EVAL_OK;
 }
 
+int func_is_null(ExprEval *ctx, RSValue *result, RSValue **argv, size_t argc, QueryError *err) {
+  VALIDATE_ARGS("isnull", 1, 1, err);
+
+  if (argv[0]->t == RSValue_Undef) {
+    if (ctx->err) {
+      QueryError_ResetError(ctx->err);
```

Comment:
Why you need this?

Reply:
b/c otherwise we would carry the error and fail later if checking for it in `RS_AddDocument` and adding `QUERY_EDOCNOTADDED`.

Reply:
What if I do something like this: IsNull(to_num(@a)) and is does not exist, You will remove the error ..

Reply:
Shell we add a condition that isnull can receive only evalProperty?